util/logs/workflow-logger.go

```diff
@@ -106,7 +103,7 @@ func NewWorkflowLogger(wfClient versioned.Interface, kubeClient kubernetes.Inter
 				scanner := bufio.NewScanner(stream)
 				for scanner.Scan() {
 					select {
-					case <-stopChan:
+					case <-ctx.Done():
```

Comment:
we were stopping the output as soon as the workflow complete, not only was this unnecessary it was stopping the logs too soon